apps/base/sops_patch.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
```

Comment:
file name should be something like:
`sops.yaml` or `sops_patch.yaml` all other files use `-` and not `_`